pkgs/development/libraries/librime/default.nix

```diff
@@ -1,20 +1,21 @@
 { lib, stdenv, fetchFromGitHub, cmake, boost, glog, leveldb, marisa, opencc,
-  libyamlcpp, gtest }:
+  libyamlcpp, gtest, capnproto, pkg-config }:
 
 stdenv.mkDerivation rec {
   pname = "librime";
-  version = "1.5.3";
+  version = "1.7.2";
 
   src = fetchFromGitHub {
     owner = "rime";
-    repo = "librime";
+    repo = pname;
     rev = version;
-    sha256 = "0xskhdhk7dgpc71r39pfzxi5vrlzy90aqj1gzv8nnapq91p2awhv";
+    sha256 = "027bav6h6k32ysxbf3jkvw5vhi6fpg1qn53f2pzpfvd0nn1s9fdh";
+    fetchSubmodules = true;
```

Comment:
Why do we need this?

Reply:
Its submodules are for dependencies, which already are done by nix. It was my fault